src/main/java/flaxbeard/immersivepetroleum/common/util/projector/Settings.java

```diff
@@ -193,11 +193,11 @@ public String toString(){
 		return "\"Settings\":[" + toNbt().toString() + "]";
 	}
 	
-	public static enum Mode{
+	public enum Mode{

```

Comment:
Weird how in some classes you switch to static, but the one class that get's re-created a lot gets its own thing :D

Reply:
Enums having static is redundant, basically

Reply:
i seee